### PR TITLE
Test Screens UI Completed

### LINKED ISSUE #14: Test screens
# REQUIREMENTS
We are going to create the following two screens

# DESIGN
COVID ALERTS V2
COVID ALERTS V2 (1)

# Acceptance Criteria
List all required acceptance criteria
- [ ] Nice UI
- [ ] Code fixed with prettier
- [ ] Navigation still working through all screens

### PR DESCRIPTION
**Pull request by vmagana3**
Co-authored-by: Mariam Rivera <riverahmariam@gmail.com>
Co-authored-by: Juan Carbajal <jjcr0397@gmail.com>

# Description
We have implemented UI type test screens,  to choose type test

  
  ## Screenshots 
image
image

### CODE REVIEW

File: COVID19/src/screens/Login.js [hunk 1]
```diff
@@ -23,7 +23,14 @@ const Login = ({navigation}) => {
...
-      const url = constans.urlAPI + '?user[email]='+formData.usuario+'&user[password]='+formData.contrasenia;
+      ///riveragmariam@gmail.com
+      //1g5717l
+      const url =
+        constans.urlAPI +
+        '?user[email]=' +
+        formData.usuario +
+        '&user[password]=' +
+        formData.contrasenia;
       //Aquí colocar antes de enviar los datos del formulario
```
Comment (fernandoamz) [inline COVID19/src/screens/Login.js:34]: Remove unnecessary comments

File: COVID19/src/screens/TestDetails.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import {View, Text} from 'react-native';
+import constans from '../utils/constans';
+import Button from '../components/Button';
+
+import styles from '../utils/styles/testDetailsStyles/testDetailsStyles';
+
+//import DateTimePicker from '@react-native-community/datetimepicker';
```
Comment (fernandoamz) [inline COVID19/src/screens/TestDetails.js:8]: Remove this unnecessary import

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by tovl1 — DISMISSED** [review]

File: COVID19/src/screens/TestDetails.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  const values = ['Positive', 'Negative'];
+  return (
+    <View style={styles.mainContainer}>
+      <Text style={styles.title}>{constans.testDetails}</Text>
+      <Text style={styles.subTitle}>{constans.selectDate}</Text>
+      <DatePicker
+        style={styles.picker}
+        date={date}
+        textColor="#5A4DCC"
+        // fadeToColor={{background: 'red'}}
```
Comment (LeyaimMagma) [inline COVID19/src/screens/TestDetails.js:26]: Remove this line

File: COVID19/src/screens/TestDetails.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  const values = ['Positive', 'Negative'];
+  return (
+    <View style={styles.mainContainer}>
+      <Text style={styles.title}>{constans.testDetails}</Text>
+      <Text style={styles.subTitle}>{constans.selectDate}</Text>
+      <DatePicker
+        style={styles.picker}
+        date={date}
+        textColor="#5A4DCC"
```
Comment (LeyaimMagma) [inline COVID19/src/screens/TestDetails.js:25]: Move this color to constants

File: COVID19/src/screens/Login.js [hunk 5]
```diff
@@ -46,9 +42,19 @@ const Login = ({navigation}) => {
...
       <View style={styles.containerContent}>
         <Text style={styles.txtTitle}>{constans.login}</Text>
-        <Input iconName="user" style={styles.input} placeholderTxt={constans.email} onChangeInput={(e) => onChange(e, 'usuario')} />
-        <Input iconName="lock" style={styles.input} placeholderTxt={constans.pw} onChangeInput={(e) => onChange(e, 'contrasenia')} />
-        <Button text={constans.login} onP={putLoginOk}/>
+        <Input
+          iconName="user"
+          style={styles.input}
+          placeholderTxt={constans.email}
+          onChangeInput={(e) => onChange(e, 'usuario')}
```
Comment (LeyaimMagma) [inline COVID19/src/screens/Login.js:49]: Add usuario to constants file

File: COVID19/src/screens/Login.js [hunk 6]
```diff
@@ -46,9 +42,19 @@ const Login = ({navigation}) => {
...
+        />
+        <Input
+          iconName="lock"
+          style={styles.input}
+          placeholderTxt={constans.pw}
+          onChangeInput={(e) => onChange(e, 'contrasenia')}
```
Comment (LeyaimMagma) [inline COVID19/src/screens/Login.js:55]: Add contraseña to constant file

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/TestDetails.js (hunk 3 above)
Comment (hearagmu) [reply COVID19/src/screens/TestDetails.js]: Fixed

File: COVID19/src/screens/Login.js (hunk 6 above)
Comment (hearagmu) [reply COVID19/src/screens/Login.js]: Fixed

**Review by LeyaimMagma — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]

**Review by mariam-rivera — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]

**Pull request merged**